epub33/core/index.html

```diff
@@ -5498,6 +5498,79 @@ <h5>Synthetic Spreads</h5>
 								of local page-progression-direction within content documents.</p>
 						</div>
 
+						<figure id="spread-none-figure">
+							<figcaption>
+								Rendering of an EPUB Fixed Layout Publication with three pages in portrait and in landscape modes,
```

Comment:
"EPUB Fixed Layout Publication" isn't a formal term of the specification. You could make this and the other below "a fixed-layout EPUB Publication" or join the pagination part to make it more concisely "Rendering of three Fixed-Layout Documents..."

But my primary comment here is whether these descriptions make sense as titles. They're trying to explain the metadata, but wouldn't it make sense to merge the figure with actual package document markup that explains the views? The title of the figure could then be reduced to something like "Rendering of Fixed Layout Documents when the EPUB Creator specifies no spreads."

Reply:
> Just to understand your proposal: let us have explicit code examples (or modify existing ones) that are in line with the figures and then cross-reference them?

I was actually thinking you could put the example markup inside the figure tags, or perhaps it would be better to add the images to an example div with package markup. Sort of like how we have some examples that show the package document markup together with content document markup (or in pub manifest I think we showed markup with internationalization examples).

Reply:
I ended up creating codes as in a usual example, and added the respective figure into the `aside`. Doing it the other way round clashed a bit with the core css, and placed the code into the center with a reduced width, which looked odd.

I left a (adapted) figure caption in, although I wonder whether it is necessary to keep them... wdyt?

Reply:
B.t.w., there is a relationship with #2087; it would be good to agree on that one and make the changes as part of this PR. I am worried of unnecessary merge complications...

Reply:
I have added a new figure for the local override example (to have a drawing for all four examples). As an aside, this also takes care of #2087.

Fix #2087